src/styles/index.scss

```diff
@@ -1,3 +1,226 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  margin: 0;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  width: 100%;
+  height: 60px;
+  display: flex;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.header__logo {
+  height: 40px;
+  width: 40px;
+}
+
+.header__nav-list {
+  margin: 0;
+  display: flex;
+  padding: 0;
+  list-style: none;
+}
+
+.header__nav-item {
+  align-items: center;
+  text-transform: uppercase;
+  margin-right: 20px;
+}
+
+.header__nav-link {
+  display: block;
+  position: relative;
+  line-height: 60px;
+  text-decoration: none;
+  color: #060b35;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.header__nav-link.is-active {
+  color: #00acdc;
+}
+
+.header__nav-item:last-child {
+  margin-right: 0;
+}
+
+.header__nav-link:hover {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  background-color: #00acdc;
+  border-radius: 8px;
+  height: 4px;
+  bottom: 0;
+  right: 0;
+  left: 0;
+}
+
+.catalog {
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+  align-items: center;
+  padding: 50px 40px;
```

Comment:
The `.catalog` class has `display: grid;` which is great for layout, but be sure to check if the `align-items` and `justify-content` properties are having the intended effect. These properties are used for aligning grid items inside the grid area and might not be necessary if you want the grid container to be the full width.